RELEASES.md

```diff
@@ -1,3 +1,174 @@
+Version 1.0.0-alpha (January 2015)
+----------------------------------
+
+  * ~2400 changes, numerous bugfixes
+
+  * Highlights
+
+    * The language itself is considered feature complete for 1.0,
+      though there is a significant amount of cleanup and bugfixes
+      remaining.
+    * Nearly 50% of the public API surface of the standard library has
+      been declared 'stable'. Those interfaces will not change.
```

Comment:
Should "will not" this be softened slightly? It seems to me we still want some room for tweaks and adjustments based on feedback.